### PR TITLE
DBZ-6883 documentation

### PR DESCRIPTION
**Pull request by jcechace**
https://issues.redhat.com/browse/DBZ-6883

First commit ought to be applied to both 2.3 and 2.4 
Second commit is exclusive to 2.4

### CODE REVIEW

**Comment by jcechace** [comment]
@jpechane Also fixed a typo and added missing configuration group in `MongoDbConnectorConfig` as part of the second commit -- I figured it's ok since it is more or less a documentation as well :)

**Review by jpechane — CHANGES_REQUESTED** [review]

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc [hunk 1]
```diff
@@ -1859,6 +1859,32 @@ A value of `0` disables this behavior.
...
+
+`fail`::
+The connector fails if the total size of change event exceed the maximum BSON size.
+
+`skip`::
+Any change events for documents exceeding the maximum (specified by the `cursor.oversize.skip.threshold` property) size will be ignored
+
+|[[mongodb-property-cursor-oversize-skip-threshold]]<<mongodb-property-cursor-oversize-skip-threshold, `+cursor.oversize.skip.threshold+`>>
+|0
+|The maximum allowed size of the stored document for which change events are processed. This includes both, the size before and after database operation, more specifically this limits the size of fullDocument and fullDocumentBeforeChange filed of MongoDB change events.
```
[the remark this answers is not in the document]
Comment (jcechace) [reply documentation/modules/ROOT/pages/connectors/mongodb.adoc:1886]: It doesn't really... The property only takes effect when `curosor.oversize.handling.mode` is set to `skip`. In such case 0 would actually skip anything larger than zero -- as I would say is expected.

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc [hunk 2]
```diff
@@ -1859,6 +1859,32 @@ A value of `0` disables this behavior.
 The value of this property must be an array of permitted https://www.mongodb.com/docs/manual/changeStreams/#modify-change-stream-output[aggregation pipeline stages] in JSON format.
 Note that this is appended after the internal pipeline used to support the connector (e.g. filtering operation types, database names, collection names, etc.).
 
+|[[mongodb-property-cursor-pipeline-order]]<<mongodb-property-cursor-pipeline-order, `+cursor.pipeline.order+`>>
+|internal_first
+|The order used to construct the effective MongoDB aggregation stream pipeline.
```
[the remark this answers is not in the document]
Comment (jcechace) [reply documentation/modules/ROOT/pages/connectors/mongodb.adoc:1864]: OK.. I will add that info

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpechane) [reply documentation/modules/ROOT/pages/connectors/mongodb.adoc:1886]: Yes, you are right, I just overcombined the fings a bit

**Comment by jpechane** [comment]
@jcechace Applied, thanks

**Pull request merged**